src/object/basic/co_integer64.c

```diff
@@ -0,0 +1,113 @@
+/******************************************************************************
+   Copyright 2022 Generac Holdings Inc.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+******************************************************************************/
+
+/******************************************************************************
+* INCLUDES
+******************************************************************************/
+
+#include "co_core.h"
+
+/******************************************************************************
+* PRIVATE DEFINES
+******************************************************************************/
+
+#define COT_ENTRY_SIZE    (uint32_t)8

```

Comment:
This caste macro is unsafe, prefer ((uint32_t)8) or ideally wrap the caste in a static inline

#define COT_ENTRY_SIZE_ (8)
static inline uint32_t COT_getEntrySize(void) {return COT_ENTRY_SIZE_ }